modules/apps/portal-workflow/portal-workflow-kaleo-api/src/main/java/com/liferay/portal/workflow/kaleo/util/comparator/KaleoDefinitionVersionModifiedDateComparator.java

```diff
@@ -48,7 +48,7 @@ public int compare(
 		Date modifiedDate1 = kaleoDefinitionVersion1.getModifiedDate();
 		Date modifiedDate2 = kaleoDefinitionVersion2.getModifiedDate();
 
-		int value = DateUtil.compareTo(modifiedDate1, modifiedDate2);
+		int value = DateUtil.compareTo(modifiedDate2, modifiedDate1);
```

Comment:
Hey @MariMalta, I don't know why we need to do this change here. I took a look at some `*ModifiedDateComparator` and everyone seems to use the same statement `DateUtil.compareTo(modifiedDate1, modifiedDate2)`. Probably the root cause may be in the above layers(DisplayContext and etc). Please check this so we can discuss more about the fix on slack.

Thanks Mari